DSE_6.8_Release_Notes.md

```diff
@@ -4,6 +4,27 @@ Components that are indicated with an asterisk (&ast;) (if any) are known to be
 
 Release notes of versions prior to 6.8.4 can be found [here](https://docs.datastax.com/en/dse/6.8/dse-admin/datastax_enterprise/releaseNotes/RNdse.html).
 
+# Release notes for 6.8.57
+7 July 2025
```

Comment:
Release date should be today (instead of the original planned date)

Reply:
Good catch! Fixed.